chat-plugins/trivia.js

```diff
@@ -1498,8 +1496,7 @@ const commands = {
 
 		if (toId(target) === 'all') {
 			if (isAccepting) {
-				for (let i = 0; i < submissionsLen; i++) {
```

Comment:
Do we still need to define `submissionsLen`?

Reply:
i removed it